src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -216,20 +187,35 @@
                                                                          (network-utils/network-list
                                                                           token-data
                                                                           network-details))
-                                              :token-display-name (:symbol token-data)))
+                                              :token-display-name (:symbol token-data)
+                                              :token-symbol       (:symbol token-data)))
               unique-owner (assoc-in [:wallet :current-viewing-account-address] unique-owner)
               entry-point  (assoc-in [:wallet :ui :send :entry-point] entry-point))
-        :fx [[:dispatch [:wallet/stop-and-clean-suggested-routes]]
-             [:dispatch
-              ;; ^:flush-dom allows us to make sure the re-frame DB state is always synced
-              ;; before the navigation occurs, so the new screen is always rendered with
-              ;; the DB state set by this event.  By adding the metadata we are omitting
-              ;; a 1-frame blink when the screen is mounted.
-              ^:flush-dom
-              [:wallet/wizard-navigate-forward
-               {:current-screen stack-id
-                :start-flow?    start-flow?
-                :flow-id        :wallet-send-flow}]]]}))))
+        :fx (if (and (not unique-owner) multi-account-balance? root-screen?)
```

Comment:
These nested `if`s also can benefit from rewriting to `cond`

Reply:
Done